examples/dataservice-read/example.cpp

```diff
@@ -129,8 +129,15 @@ int RunExample() {
   std::shared_ptr<olp::http::Network> http_client = olp::client::
       OlpClientSettingsFactory::CreateDefaultNetworkRequestHandler();
 
-  // Initialize authentication settings
-  olp::authentication::Settings settings({kKeyId, kKeySecret});
+  // Get your access credentials from the **credentials.properties** file that
+  // you can download from the
```

Comment:
```suggestion
  // you can download from the Open Location Platform (OLP)
```

Reply:
In the previous review, I asked you to shorten the link text, not to delete "Open Location Platform (OLP)". Sorry if I misled you in any way :)

Reply:
Done